CHANGELOG.md

```diff
@@ -9,19 +9,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `mypy` for search space and objectives
 - Class hierarchy for objectives
 - Deserialization is now also possible from optional class name abbreviations
-- `Kernel` base class allowing to specify kernels
-- `MaternKernel` class can be chosen for GP surrogates
-- `hypothesis` strategies and roundtrip test for kernels, constraints, objectives, priors
-  and acquisition functions
+- `Kernel`, `MaternKernel`, and `ScaleKernel`  classes for specifying kernels
+- `hypothesis` strategies and roundtrip test for kernels, constraints, objectives,
+  priors and acquisition functions
 - New acquisition functions: `qSR`, `qNEI`, `LogEI`, `qLogEI`, `qLogNEI`
 - Serialization user guide
 - Basic deserialization tests using different class type specifiers
 - `GammaPrior`, `HalfCauchyPrior`, `NormalPrior`, `HalfNormalPrior`, `LogNormalPrior`
   and `SmoothedBoxPrior` can now be chosen as lengthscale prior
+- `KernelFactory` protocol enabling flexible construction of kernels
```

Comment:
Hi @AVHopp, currently getting a doc error due to missing reference. Could this be related to `Protocol` in some way?

Reply:
Reason is that some reference in a docstring is not found (have not looked into this, but this is the error):
<img width="951" alt="image" src="https://github.com/emdgroup/baybe/assets/84016733/381ea303-2b24-4a69-960e-2da0f65e6b7f">

Reply:
Yes, I saw that, too, but I don't understand what is the problem really ... 🤔

Reply:
The corresponding kernel field in the factory has no docstring: 
<img width="638" alt="image" src="https://github.com/emdgroup/baybe/assets/84016733/f0d94742-3718-4c84-90f8-d88889857bec">

Reply:
Eagle eyes!!! Thanks, would have searched for decades :D